packages/api-request-builder/package.json

```diff
@@ -19,9 +19,15 @@
   "bugs": "https://github.com/commercetools/nodejs/issues",
   "license": "MIT",
   "author": "Nicola Molinari <nicola.molinari@commercetools.com> (https://github.com/emmenko)",
-  "main": "dist/api-request-builder.cjs.js",
-  "module": "dist/api-request-builder.es.js",
-  "browser": "dist/api-request-builder.umd.js",
+  "main": "dist/commercetools-api-request-builder.cjs.js",
+  "module": "dist/commercetools-api-request-builder.esm.js",
+  "umd:main": "dist/commercetools-api-request-builder.umd.min.js",
+  "preconstruct": {
+    "entrypoints": [
+      "./index.js"
+    ],
+    "umdName": "commercetoolsApiRequestBuilder"
+  },
```

Comment:
The naming change is transparent to users it follows the preconstruct and general community standards.